TipsForSlack.md

```diff
@@ -2,17 +2,22 @@
 ## Tips to make your Slack experience fun & fruitful
 * Join & participate in public channels! There is everything from #interview-questions to #personal-finance to #feelings to #python and many more.
 * Job opportunities can be posted in [#jobs](https://theadanetwork.slack.com/messages/jobs/).
-* Use the channel that is most appropriate & only cross-post when relevant. An example of when you shouldn't cross post is posting a message of "I'm selling my bike" to both #free-or-selling and #general (it should only be posted in #free-or-selling). An example of when you could cross post might be posting in both #freelancing & #jobs. 
+* Use the channel that is most appropriate & only cross-post when relevant. An example of when you shouldn't cross post is posting a message of "I'm selling my bike" to both #free-or-selling and #random (it should only be posted in #free-or-selling). An example of when you could cross post might be posting in both #freelancing & #jobs. 
 * Don't spam channels with links. Share what you found interesting or what your opinion is when you post. Check to see if someone has already posted the article, video, tweet, etc before you. If someone already has shared the thing you wanted to share, start a conversation!
 * Don't be afraid to DM! Slack is an excellent way to connect with other Ada alums who you might not already know. 
 
+## The #general channel
+The #general channel is a group-wide channel to which you will automatically be added upon joining slack. This channel is restricted: since everyone is in this channel and you can't leave the channel, posting is limited to members of the Community Team for the purpose of making announcements and letting members know about community updates. Miscellaneous chit chat should take place in the #random channel.
```

Comment:
Did we want to say something about if someone has an all-adies announcement appropriate for #general to message us in slack and ask if one of us can post it? I might give a couple examples to set people on the right track. “Announcing that a new cohort of Adies will soon be joining us or <other sample announcement> are examples of things that might be posted in general.”

Reply:
How about something like, "Announcements for General would be things like when a new cohort is about to join the slack or if the code of conduct changes. If you have something you'd like to share with everyone announcement style, please message someone on the community team and we'll help you get it out there."

Reply:
Excellent examples--thanks. Adding in now!